oeps/oep-0038-Data-Modeling.rst

```diff
@@ -0,0 +1,419 @@
+======================================
+OEP-38:  Data Modeling Best Practices
+======================================
+
++---------------+------------------------------------------------------------+
+| OEP           | :doc: `OEP-0038 <oep-0038-data-modeling>`                  |
++---------------+------------------------------------------------------------+
+| Title         | Data Modeling Best Practices                               |
++---------------+------------------------------------------------------------+
+| Last Modified | 2019-09-19                                                 |
++---------------+------------------------------------------------------------+
+| Author        | Brian Beggs <bbeggs@edx.org>                               |
++---------------+------------------------------------------------------------+
+| Arbiter       | Albemarle                                                  |
++---------------+------------------------------------------------------------+
+| Status        | Under Review                                               |
++---------------+------------------------------------------------------------+
+| Type          | Best Practices                                             |
++---------------+------------------------------------------------------------+
+| Created       | 2019-05-01                                                 |
++---------------+------------------------------------------------------------+
+| Review Period | 2019-08-01 - 2019/10/01                                    |
++---------------+------------------------------------------------------------+
+| Resolution    | 'Original pull request'                                    |       
++---------------+------------------------------------------------------------+
+
+Abstract
+========
+
+In order to obtain the highest possible value from data collected in the
+Open edX ecosystem, this document attempts to provide guiding thoughts and
+principles on data modeling.
```

Comment:
This OEP appears to be "hard-wrapped", it would be better to use logical breaks and allow auto-wrapping; see [OEP-19 Best Practices](https://open-edx-proposals.readthedocs.io/en/latest/oep-0019-bp-developer-documentation.html#best-practices)